Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,16 @@
 import React from 'react';
 import './App.css';
 
+import CardContainer from "./components/CardComponents/CardContainer";
+import HeaderContainer from "./components/HeaderComponents/HeaderContainer";
+import FooterContainer from "./components/FooterComponents/Footer";
+
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+      <HeaderContainer />
```

Comment:
Love this. Whenever logically possible always try to make your root file like app.js as clean and simple as possible like this where all you're doing is importing things and thus have all your code and logic in the correct and relevant components which is the whole point of react and makes your code cleaner, more organized and easier to debug